src/DSi.cpp

```diff
@@ -864,15 +943,19 @@ u16 ARM9Read16(u32 addr)
 
     return NDS::ARM9Read16(addr);
 }
+#pragma GCC push_options
+#pragma GCC optimize ("O0")
```

Comment:
what is this for?

Reply:
Good catch.  That is a debugging remainder. I will remove it. Thought i caught them all.